projects/plugins/social/src/js/components/connection-screen/index.js

```diff
@@ -20,42 +42,62 @@ const ConnectionScreen = () => {
 			registrationNonce: store.getRegistrationNonce(),
 		};
 	} );
+
+	const {
+		userIsConnecting,
+		siteIsRegistering,
+		handleRegisterSite,
+		registrationError,
+	} = useConnection( {
+		from: 'jetpack-social',
+		redirectUri: 'admin.php?page=jetpack-social',
+		...connectProps,
+	} );
+
 	return (
-		<ConnectScreen
-			buttonLabel={ __( 'Connect Jetpack Social', 'jetpack-social' ) }
-			pricingTitle={ __( 'Jetpack Social', 'jetpack-social' ) }
-			title={ __( 'Social Media Automation for WordPress Sites', 'jetpack-social' ) }
-			from="jetpack-social"
-			redirectUri="admin.php?page=jetpack-social"
-			{ ...connectProps }
-		>
-			<h3>
-				{ __(
-					'Share your site’s posts on several social media networks automatically when you publish a new post',
-					'jetpack-social'
-				) }
-			</h3>
-			<ul>
-				<li>
-					{ __(
-						'Reach your maximum potential audience, not just those who visit your site',
-						'jetpack-social'
-					) }
-				</li>
-				<li>
-					{ __(
-						'Be found by prospective readers or customers on their preferred social site or network',
-						'jetpack-social'
-					) }
-				</li>
-				<li>
-					{ __(
-						'Allow people who like your content to easily share it with their own followers, giving you even greater visibility',
-						'jetpack-social'
-					) }
-				</li>
-			</ul>
-		</ConnectScreen>
+		<Dialog
+			className={ styles.card }
+			primary={
+				<>
+					<div className={ styles.column }>
+						<ProductOffer
+							className={ styles.offer }
+							slug={ 'jetpack-social' }
+							title={ 'Jetpack Social' }
+							subTitle={ __(
+								'Share your posts with your social media network and increase your site’s traffic',
+								'jetpack-social'
+							) }
+							features={ [
+								'Connect with Twitter, Facebook, LinkedIn and Tumblr',
+								'Select the social media to share posts while publishing',
+								'Publish custom messages',
```

Comment:
we need to translate these string:

```suggestion
								__( 'Connect with Twitter, Facebook, LinkedIn and Tumblr', 'jetpack-social' ),
								__( 'Select the social media to share posts while publishing', 'jetpack-social' ),
								__( 'Publish custom messages', 'jetpack-social' ),
```

Reply:
I'll make a follow-up PR that gets this data from My Jetpack instead 👍🏼